config/overlay.in

```diff
@@ -0,0 +1,68 @@
+# The component directory name
```

Comment:
1. Can't this file be auto-generated, as the rest of package fragments?
2. Can you give an example of a .config file for a "combined" release of ct-ng and an overlay? I am curious how the source location for an overlay looks like, is it a relative/absolute path? If relative, what is it relative to?

Reply:
> Can't this file be auto-generated, as the rest of package fragments?
Don't know. Probably I haven't found a way to do it the first time around. Let me look at it some more.

> Can you give an example of a .config file for a "combined" release of ct-ng and an overlay?
https://github.com/jcmvbkbc/crosstool-NG/blob/xtensa-1.22.x/samples/xtensa-lx106-elf/crosstool.config#L9